flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/UserDefinedFunctionHelper.java

```diff
@@ -215,6 +216,38 @@ public static void validateClass(Class<? extends UserDefinedFunction> functionCl
 		validateClass(functionClass, true);
 	}
 
+	/**
+	 * Validates a {@link UserDefinedFunction} class for usage in the runtime.
+	 *
+	 * <p>Note: This is for the final validation when actual {@link DataType}s for arguments and result
```

Comment:
just a nitpick but you can write `{@link DataType DataTypes}`, then it will link to `DataType` but the link will be called `DataTypes`.

Reply:
thanks for the hint, actually I'm aware of this trick. but I find this one more readable in the code.